modules/serverless-build-func-kn.adoc

```diff
@@ -6,30 +6,16 @@
 [id="serverless-build-func-kn_{context}"]
 = Building functions
 
-Before you can run a function, you must build the function project by using the `kn func build` command. The build command reads the `func.yaml` file from the function project directory to determine the image name and registry.
+Before you can run a function, you must build the function project by using the `kn func build` command. This command creates an OCI container image that can be run locally on your computer, or on a Kubernetes cluster. The build command uses the function project name and the image registry name to construct a fully qualified image name for your function.
```

Comment:
NBD but you don't need a comma before `or on a Kubernetes cluster.`

Reply:
Good catch, also updated Kubernetes -> {product-title} since these are product docs 😉